package/network/utils/ltq-dsl-base/files/lib/functions/lantiq_dsl.sh

```diff
@@ -510,6 +518,12 @@ errors() {
 		echo "dsl.errors_crc_p_far=${crc_pf:-nil}"
 		echo "dsl.errors_crcp_p_near=${crcp_pn:-nil}"
 		echo "dsl.errors_crcp_p_far=${crcp_pf:-nil}"
+		echo "dsl.errors_rxretr_near=${rxretn:-nil}"
+		echo "dsl.errors_txretr_near=${txretn:-nil}"
+		echo "dsl.errors_rxcorr_near=${rxcorn:-nil}"
+		echo "dsl.errors_rxretr_far=${rxretf:-nil}"
+		echo "dsl.errors_txretr_far=${txretf:-nil}"
+		echo "dsl.errors_rxcorr_far=${rxcorf:-nil}"
```

Comment:
I would also output the new values on a "dslstat" action.

Reply:
I mean, you should also add the output of the new values in the "else" path below.

Reply:
Clear, Info added for those stats now. Tested locally and looks fine.